### DISCUSSION TITLE
How do i connect to discord rich presence?

### DISCUSSION THREAD

**Discussion by Metascov** [pear-devs/pear-desktop#1137] [Q&A]
Hello
I'm super inexperienced with code and stuff, but just wanted to connect to my discord for rich presence. How do I do that?
![image](https://github.com/th-ch/youtube-music/assets/133183784/bdb33eae-a172-41d1-8550-3612a584a9d2)

Thanks in advance!

**Comment by thijsi123** [comment]
![image](https://github.com/th-ch/youtube-music/assets/64159148/bff16879-d017-4540-b04c-fc9d3ceab893)
make sure that your (game)activity status is turned on.
